### PR TITLE
Considering customer ignore_features

### PR DESCRIPTION
**Pull request by joker2411**
## Description of the change

> Including ignore_features while fetching input_column_types.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/predictions/rudderstack_predictions/connectors/SnowflakeConnector.py [hunk 1]
```diff
@@ -749,7 +749,7 @@ def validate_class_proportions(
         if len(no_invalid_rows) > 0:
             error_msg = ""
             for row in result_table:
-                error_msg += f"\t{row[label_column]} - user count:  {row['COUNT']} ({100*row['NORMALIZED_COUNT']:.2f}%)\n"
+                error_msg += f"\t{row[label_column.upper()]} - user count:  {row['COUNT']} ({100*row['NORMALIZED_COUNT']:.2f}%)\n"
```
Comment (dpatchigolla) [inline src/predictions/rudderstack_predictions/connectors/SnowflakeConnector.py:752]: Was it creating an error earlier?

File: src/predictions/rudderstack_predictions/connectors/Connector.py [hunk 2]
```diff
@@ -42,13 +44,22 @@ def get_input_column_types(
...
             "numeric": numeric_columns,
             "categorical": categorical_columns,
             "arraytype": arraytype_columns,
             "timestamp": timestamp_columns,
         }
 
+        for key in input_column_types:
+            input_column_types[key] = [
+                value
+                for value in input_column_types[key]
```
Comment (dpatchigolla) [inline src/predictions/rudderstack_predictions/connectors/Connector.py:57]: Suggestion: Replace `for key in input_column_types:` with
`for column_type, columns in input_column_types.items():`

**Review by dpatchigolla — APPROVED** [review]
Looks good. minor naming suggesitons.

File: src/predictions/rudderstack_predictions/connectors/SnowflakeConnector.py (hunk 1 above)
Comment (joker2411) [reply src/predictions/rudderstack_predictions/connectors/SnowflakeConnector.py:752]: yes. Column names in snowflake are in uppercase. So, if we give smallcase label names in configs, then it was throwing an error.

**Pull request merged**